docs/source/raft_configuration.md

```diff
@@ -86,22 +86,28 @@ other, and TLS certificates issued by a public TLS CA for the client facing API.
 
   * `ClientCertificate`, `ClientPrivateKey`: The file path of the client TLS certificate
   and corresponding private key.
-  * `ListenPort`: The port the cluster listens on. If blank, the port is the same
-  port as the orderer general port (`general.listenPort`)
+  * `ListenPort`: The port the cluster listens on. 
+  It must be same as `consenters[i].Port` in Channel configuration. 
+  If blank, the port is the same port as the orderer general port (`general.listenPort`)
   * `ListenAddress`: The address the cluster service is listening on.
   * `ServerCertificate`, `ServerPrivateKey`: The TLS server certificate key pair
   which is used when the cluster service is running on a separate gRPC server
   (different port).
-  * `SendBufferSize`: Regulates the number of messages in the egress buffer.
 
 Note: `ListenPort`, `ListenAddress`, `ServerCertificate`, `ServerPrivateKey` must
 be either set together or unset together.
 If they are unset, they are inherited from the general TLS section,
 in example `general.tls.{privateKey, certificate}`.
+In the use case when general TLS is disabled, these parameters must be set:
+ - use different `ListenPort` than the orderer general port
+ - Properly configure TLS root CAs. Search scope includes
```

Comment:
Why would we need to configure the TLS root CAs when TLS is disabled? I've never tried to use the orderer without TLS enabled but this seems odd.

Reply:
I imagine this is if you aren't defining the TLS CAs in the channel config? Although seems better to define them in the channel config even if TLS is disabled generally.

Reply:
I see consenters mutual connection errors when TLS root CAs could not be found in anywhere. I am not 100% sure of the search scope, but indeed we should have at least once.

Reply:
How about re-statement as  `Properly configure TLS root CAs. For instance, define them for each orderer organization in channel config ` @jyellick

Reply:
Sorry for losing track of this.

Yes, that sentence is better.  Though I think it would be superior to say "Properly configure TLS root CAs in the channel configuration."  I do not believe that recent versions of Fabric will even allow you to define Raft endpoints if their TLS certificate is not valid according to some TLS CA in the channel config.